### ISSUE TITLE
fairgame puts incorrect captcha and gets my account temp-banned

### ISSUE THREAD

**Issue by AlphaConn** [Hari-Nagarajan/fairgame#811]
fairgame fails to put correct captcha 

**To Reproduce**

run for a few hours untill amazon prompts captcha
bot will attempt to solve
then gets it wrong every time lately

**Expected behavior**

used to do this with no issue now it fails

**Screenshots**
If applicable, add screenshots to help explain your problem. These are found in the bot directory.
screenshot of log is attached to show after it fails it cannot look at listings because amazon will temp-ban because of this

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser google chrome
 - Version not sure

**Additional context**
basically bot gets captcha wrong and gets account temp-banned

![Screenshot (55)](https://user-images.githubusercontent.com/87639475/126126497-bfe82332-4dd7-48f8-90e3-ff96a178e212.png)
